tests/lkft/ltp/ltp_cmdfile

```diff
@@ -1 +1 @@
-can,cap_bounds,commands,connectors,containers,controllers,cpuhotplug,crypto,cve,dio,dma_thread_diotest,fcntl-locktests,filecaps,fs,fs_bind,fs_perms_simple,fs_readonly,fsx,hugetlb,hyperthreading,ima,input,io,io_cd,io_floppy,ipc,kernel_misc,ltp-aiodio.part1,ltp-aiodio.part2,ltp-aiodio.part3,ltp-aiodio.part4,ltp-aio-stress.part1,ltp-aio-stress.part2,math,mm,net.features,net.ipv6,net.ipv6_lib,net.multicast,net.nfs,net.rpc,net.rpc_tests,net.sctp,net_stress.appl,net_stress.broken_ip,net_stress.interface,net_stress.ipsec_dccp,net_stress.ipsec_icmp,net_stress.ipsec_sctp,net_stress.ipsec_tcp,net_stress.ipsec_udp,net_stress.multicast,net_stress.route,net.tcp_cmds,net.tirpc_tests,nptl,numa,pty,sched,scsi_debug.part1,securebits,smack,smoketest,syscalls,syscalls-ipc,tpm_tools,tracing,uevent
+can,cap_bounds,commands,connectors,containers,controllers,cpuhotplug,crypto,cve,dio,dma_thread_diotest,fcntl-locktests,filecaps,fs,fs_bind,fs_perms_simple,fs_readonly,fsx,hugetlb,hyperthreading,ima,input,io,io_cd,io_floppy,ipc,kernel_misc,ltp-aiodio.part2,ltp-aiodio.part4,math,mm,net.features,net.ipv6,net.ipv6_lib,net.multicast,net.nfs,net.rpc,net.rpc_tests,net.sctp,net_stress.appl,net_stress.broken_ip,net_stress.interface,net_stress.ipsec_dccp,net_stress.ipsec_icmp,net_stress.ipsec_sctp,net_stress.ipsec_tcp,net_stress.ipsec_udp,net_stress.multicast,net_stress.route,net.tcp_cmds,net.tirpc_tests,nptl,numa,pty,sched,scsi_debug.part1,securebits,smack,smoketest,syscalls,syscalls-ipc,tpm_tools,tracing,uevent
```

Comment:
@bobolmw , could you please add comments in ltp_cmdfile, why "ltp-aiodio.part1,ltp-aiodio.part3,,ltp-aio-stress.part1,ltp-aio-stress.part2" are skipped?